WickedEngine/ArchiveVersionHistory.txt

```diff
@@ -1,5 +1,6 @@
 This file contains changelog of wi::Archive versions
 
+89: serialized WeatherComponent::atmosphereParameters: rayMarchMinMaxSPP, distanceSPPMaxInv and aerialPerspectiveScale
```

Comment:
We don't need to increase archive version number for things in scene serialization any more. The component managers have their own version number now which is preferred usage now (where they are declared in the Scene, you can specify version number in ComponentLibrary::Register). But this will work too, just not the best place. If you use the version number of component, then instead `archive.GetVersion()` you will need to use `seri.GetVersion()`

Reply:
Oh I see. I wondered why Register with a version number, I'll make change

Reply:
Would this need an entry in archive history?

Reply:
And this doesn't need tracking in ArchiveVersion.txt. Since the changes won't be scattered across codebase and be local to a component serializing, it will be easy to find changes of component versions just in the code.

Reply:
Should be good now!